### PR TITLE
fix tm-forum api

### PR DESCRIPTION
**Pull request by dleonardi95**
Some configs was missing, those changes should solve the deploy for the tmforum in both Dev and Sbx enviroment

### CODE REVIEW

File: ionos_dev/marketplace/tmforum-api/values.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        value: "false"
+
+  # -- list of apis to be deployed
+  # -- every api needs to define a name and image. Basepath is required when routes or ingress will be enabled. Beside that, they can overwrite the default-config by using the same keys.
+  # -- be aware: when you change the image repositrory or the tag for an api, you have to provide both values for the changes to take effect
+  apis:
+    - name: party-catalog
+      image: tmforum-party-catalog
+      basePath: /
+      specUrl: https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.0.0/swagger/TMF632-Party-v4.0.0.swagger.json
```
Comment (fdelavega) [inline ionos_dev/marketplace/tmforum-api/values.yaml:67]: Is the specUrl still used? the swagger file is now part of the TMForum API project

**Comment by wistefan** [comment]
This is just the documentation of the apis that was deployed for demo. There is no point in deploying it here.

**Comment by wistefan** [comment]
@bialask-daiteap I dont understand you question. But still, this is nothing that should be deployed anywhere else then demo and it will be replaced by the access-node chart anyways. I would recommend to delete that PR.

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]
Please split MR one per env

**Pull request closed without merging**